requirements.txt

```diff
@@ -1,7 +1,38 @@
+appdirs==1.4.4
 asgiref==3.3.1
 certifi==2020.12.5
+cffi==1.14.5
+chardet==4.0.0
+cryptography==3.4.6
+cycler==0.10.0
+distlib==0.3.1
 Django==3.1.7
+django-aesfield==3.0
```

Comment:
I got an error doing
`pip install -r requirements.txt`

> ERROR: Cannot install -r requirements.txt (line 10) and Django==3.1.7 because these package versions have conflicting dependencies.
> 
> The conflict is caused by:
>     The user requested Django==3.1.7
>     django-aesfield 3.0 depends on Django<=2.1 and >=1.11
> 
> To fix this you could try to:
> 1. loosen the range of package versions you've specified
> 2. remove package versions to allow pip attempt to solve the dependency conflict

Is django-aesfield being used right now? If not, we should remove and get a package that fits our Django version.

Reply:
The django-aes field might be used in the Schools app. but requires a downgrade.